blocks/init/components/scroll-to-top/assets/index.js

```diff
@@ -0,0 +1,11 @@
+const selector = '.js-scroll-to-top';
+
+if ($(selector).length) {
```

Comment:
Is this jquery? I thought we abandoned it :eyes:

Reply:
Yea, it is, but this is copied form other projects. I will leave this to refactor for some volunteer

Reply:
There are 2 options already implemented in A1, one with smooth-scroll library, one native though with native, top only needs to be 0 in the code, instead of element
And with native for IE, compatibility is unknown, all else is green.
https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll

https://github.com/infinum/wordpress-a1portal-careers/blob/master/wp-content/themes/a1-careers-page/src/blocks/assets/scripts/utils/scroll.js
https://github.com/infinum/wordpress-a1portal-careers/blob/master/wp-content/themes/a1-careers-page/src/blocks/assets/scripts/components/scroll-to-top.js